### PR TITLE
Integrate patch set

### PR DESCRIPTION
**Pull request by mkholjuraev**
This PR will add:

-  patch set info on Inventory detail header.
-  status and patch status filters into Review systems table in the patch set wizard.
-  default filters into Review systems table  in the patch set wizard.
-  patch set column into Review systems table  in the patch set wizard.
- remove feature flags for the Patch set table.

### CODE REVIEW

**Review by gabipodolnikova — COMMENTED** [review]
Found just one tiny thing and these are just a few mostly UX thoughts:
In the Inventory detail header, when there is no patch set name or it is undefined, you see a blank space or undefined - I would consider some label (or something) to have it clearly stated that it is not a mistake that there is a blank space.
The same applies to the System table in the Create patch set wizard. There is a blank column.
Other thing in this table is that I would consider some default sorting, because when you select a system and click Next and then go Back you still have a selected system, but because there is no sorting, the selected system is in a different spot and I was confused.

File: src/SmartComponents/SystemDetail/InventoryDetail.js [hunk 1]
```diff
@@ -30,8 +30,8 @@ const InventoryDetail = ({ match }) => {
         ({ entityDetails }) => entityDetails && entityDetails.entity
     );
 
-    const hasThirdPartyRepo = useSelector(
-        ({ entityDetails }) => entityDetails && entityDetails.hasThirdPartyRepo
+    const { hasThirdPartyRepo, patchSetName } = useSelector(
+        ({ entityDetails }) => entityDetails ? entityDetails : {}
```
[the remark this answers is not in the document]
Comment (mkholjuraev) [reply src/SmartComponents/SystemDetail/InventoryDetail.js:34]: Regarding the the missing patch set name, it was actually UX design :). I would also opt for having something there.

Nice suggestion to simplify into `entityDetails ?? {}`. I am happy to adop it.

**Comment by mkholjuraev** [comment]
@gabipodolnikova one review would be great.

**Review by gabipodolnikova — APPROVED** [review]
Looking good!
Thanks for explaining the missing name, I think it would be great to talk about it with UX, also mention the sorting thing in the table (in the wizard) please, it bothered me a little bit :D

**Comment by mkholjuraev** [comment]
@gabipodolnikova Thanks for the review!

I will definately discuss it. Can you please describe sorting issue? is it Patch set column sort?

**Pull request merged**